src/components/Video/index.js

```diff
@@ -52,11 +52,8 @@ export default class Video extends Component {
   watch = () => {
     logEvent('button', 'video')
     this.setState({ watching: true })
-    this.play()
   }
 
-  play = () => {}
```

Comment:
do you know why was it there in the first place? )

Reply:
I think once upon a time here was another logic (or it was planned =)). But for now it looks like unnecessary code, so I just removed it.

Reply:
probably some GA event that was remove along these refactoring. It's fine for now. 👍